source/basic.tex

```diff
@@ -1692,8 +1692,7 @@
 to be considered.
 The sets of namespaces and entities
 are determined entirely by
-the types of the function arguments
-(and the namespace of any template template argument).
+the types of the function arguments.
```

Comment:
If we're killing words here, take the next sentence as well. The moons of Saturn also don't contribute to the sets of associated namespaces and entities, but we don't need to proclaim that fact normatively.

Reply:
The parenthetical was nonsensical; the next sentence is merely redundant. I wouldn't object to removing the next sentence, but maybe I should open a completely orthogonal PR for that.